do_cmake.sh

```diff
@@ -29,7 +29,7 @@ if [ -r /etc/os-release ]; then
       rhel|centos)
           MAJOR_VER=$(echo "$VERSION_ID" | sed -e 's/\..*$//')
           if [ "$MAJOR_VER" -ge "8" ] ; then
-              PYBUILD="3.6"
+              PYBUILD="3.9"
```

Comment:
what if some of us are still building ceph using this script on RHEL8. if that's the case, shall we have something like
```sh
if [ "$MAJOR_VER" -ge "9" ] ; then
  PYBUILD="3.9"
elif [ "$MAJOR_VER" -ge "8" ] ; then
  PYBUILD="3.6"
fi
```